tasks/app/views/people/showtasks.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1><%= @person.name %></h1>
+<div class="center">
+  <span>
+    To Do:
+    <br />
+    <% @tasks.each do |task| %>
```

Comment:
The code within this loop through `@tasks` is very similar to the code in the loop later on in this template that also goes through `@tasks`. How might we be able to effectively DRY this out?